azure-quantum/azure/quantum/qiskit/results/resource_estimator.py

```diff
@@ -6,7 +6,63 @@
 To install run: pip install azure-quantum[qiskit]"
     )
 
+import markdown
+
 class ResourceEstimatorResult(Result):
+    def __init__(
+        self,
+        **kwargs
+    ) -> None:
+        super().__init__(**kwargs)
+        self.summary = ResourceEstimatorResultSummary(**kwargs)
+
+    def _repr_html_(self):
+        return self._repr_html_success_() if self.success else self._repr_html_error_()
+
+    def _repr_html_success_(self):
+        html = ""
+
+        md = markdown.Markdown(extensions=['mdx_math'])
+        for group in self.data()['reportData']['groups']:
+            html += f"""
+                <details {"open" if group['alwaysVisible'] else ""}>
+                    <summary style="display:list-item">
+                        <strong>{group['title']}</strong>
+                    </summary>
+                <table>"""
```

Comment:
If this is markup, is there a reason to use `<table>` elements instead of markup?
```
| h1 | h2 |
|----|----|
| v1 | v2 |
```

Reply:
The table is making use of HTML features that cannot be accessed through Markdown